NotadogApi/Controllers/UsersController.cs

```diff
@@ -38,22 +38,33 @@ public UsersController(IUserService userService, IJwtTokenGenerator jwtTokenGene
         /// Get authentification token.
         /// </summary>  
         [HttpPost("login")]
-        public async Task<IActionResult> Login(UserCredentials credentials)
+        public async Task<IActionResult> Login(UserLoginCredentials credentials)
         {
             var user = await _userService.GetOneByEmailAsync(credentials.Email);
 
             if (user == null)
             {
-                return BadRequest(HttpStatusCode.Forbidden);
+                return NotFound();
             }
 
             if (user.Password != credentials.Password)
             {
-                return BadRequest(HttpStatusCode.Forbidden);
+                return Unauthorized();
             }
 
-            var Token = await _jwtTokenGenerator.CreateToken(user.Id);
-            return Ok(Token);
+            var token = await _jwtTokenGenerator.CreateToken(user.Id);
+            return Ok(token);
+        }
+
+        /// <summary>
+        /// Get authentification t.
+        /// </summary>  
+        [HttpPost("signup")]
+        public async Task<IActionResult> Signup(UserSignupCredentials credentials)
+        {
+            var user = await _userService.CreateAsync(credentials.Name, credentials.Email, credentials.Password);

```

Comment:
Here will be exception, if **Email** is busy. I suggest 3 ways to take into consideration this case:
- catch exception here;
- catch exception in service and return null;
- check if user with **Email** already exists and return null.
P.S. Looks like it's time to write something in service ;)